cloudinit/config/cc_ssh_authkey_fingerprints.py

```diff
@@ -57,10 +57,35 @@ def _gen_fingerprint(b64_text, hash_meth='sha256'):
         return '?'
 
 
+# When updating the list, also update the VALID_KEY_TYPES list in
+# cloudinit/ssh_util.py
```

Comment:
Should we instead import `VALID_KEY_TYPES` from `ssh_util` and use it here?

Reply:
You thinking of just having the list in `ssh_util`, and then fetching it from there?
That should work, plus, it removes double lists.
I gotta experiment, my python isn't that good

Reply:
It's now importing `VALID_KEY_TYPES` from `ssh_util`